src/relayFeeCalculator/chain-queries/boba.ts

```diff
@@ -11,7 +15,7 @@ export class BobaQueries implements QueryInterface {
   private spokePool: OptimismSpokePool;
 
   constructor(
-    provider: providers.Provider,
+    private provider: providers.Provider,
```

Comment:
out of curiosity, why make it private?

Reply:
I tend toward encapsulating the data - however, upon further review, the master branch does use `readonly` for several providers. I've pushed a commit that has a consistent `readonly` modifier on all `provider` instance variables.